softwerkskammer/lib/mailsender/mailsenderService.js

```diff
@@ -101,19 +103,25 @@ module.exports = {
     return activitiesService.getActivityWithGroupAndParticipants(activityURL, function (err, activity) {
       if (err) { return callback(err, statusmessageForError(type, err)); }
       message.setBccToMemberAddresses(activity.participants);
+      message.setIcal(icalService.activityAsICal(activity).toString());
       sendMail(message, type, callback);
     });
   },
 
-  sendMailToInvitedGroups: function (invitedGroups, message, callback) {
+  sendMailToInvitedGroups: function (invitedGroups, activityURL, message, callback) {
     var type = '$t(mailsender.invitation)';
     return groupsService.getGroups(invitedGroups, function (err, groups) {
       if (err) { return callback(err, statusmessageForError(type, err)); }
       if (groups.length === 0) { return callback(null, statusmessageForError(type, new Error('Keine der Gruppen wurde gefunden.'))); }
       async.map(groups, groupsAndMembersService.addMembersToGroup, function (err1, groups1) {
         if (err1) { return callback(err1, statusmessageForError(type, err1)); }
         message.setBccToGroupMemberAddresses(groups1);
-        sendMail(message, type, callback);
+        activitystore.getActivity(activityURL, function (err2, activity) {
+          if (activity) {
+            message.setIcal(icalService.activityAsICal(activity).toString());
```

Comment:
Werde ich als boyscout beim nächsten PR mitschieben. 

> Am 05.09.2016 um 20:59 schrieb Nicole Rauch <notifications@github.com>:
> 
> In softwerkskammer/lib/mailsender/mailsenderService.js:
> 
> >      var type = '$t(mailsender.invitation)';
> >      return groupsService.getGroups(invitedGroups, function (err, groups) {
> >        if (err) { return callback(err, statusmessageForError(type, err)); }
> >        if (groups.length === 0) { return callback(null, statusmessageForError(type, new Error('Keine der Gruppen wurde gefunden.'))); }
> >        async.map(groups, groupsAndMembersService.addMembersToGroup, function (err1, groups1) {
> >          if (err1) { return callback(err1, statusmessageForError(type, err1)); }
> >          message.setBccToGroupMemberAddresses(groups1);
> > -        sendMail(message, type, callback);
> > +        activitystore.getActivity(activityURL, function (err2, activity) {
> > +          if (activity) {
> > +            message.setIcal(icalService.activityAsICal(activity).toString());
> Ich finde groß besser
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>